### PR TITLE
Attendees model integrated into database schema

### LINKED ISSUE #2: Create database tables (users, attendees, events) ~ 2 hrs
Attendees table, consisting of a list of users associated with an event.

### PR DESCRIPTION
**Pull request by Tek58**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -27,4 +28,10 @@ model User {
   events        Event[]
 }
 
-
+model Atendees {
+  id            Int       @id @default(autoincrement())
+  name          String?
+  event_name    Event?    @relation(fields: [eventId], references: [id])
```
Comment (simbamentor) [inline prisma/schema.prisma]: can you call this `event` instead?

let's stay by this convention: "if you have a relationship let's call it by the table name"

File: prisma/schema.prisma [hunk 2]
```diff
@@ -27,4 +28,10 @@ model User {
...
+  eventId       Int?
+  email         String?   @unique
```
Comment (simbamentor) [inline prisma/schema.prisma]: It's not needed that this field is unique here. We can create new attendees with the same email addresses.

In the future we will have more fields on the attendee table, like for example "Accepted Event". So when there are multiple events it's possible that you've accepted one event but not the other. It's also possible that you use a different name on one event vs on the other event

File: prisma/schema.prisma (hunk 1 above)
Comment (simbamentor) [reply prisma/schema.prisma]: having a different name should be the exception

**Review by simbamentor — COMMENTED** [review]
added some comments

File: prisma/schema.prisma (hunk 2 above)
Comment (Tek58) [reply prisma/schema.prisma]: changes implemented

**Pull request merged**